docs/contributing.rst

```diff
@@ -1,3 +1,19 @@
+Getting involved
```

Comment:
Can you pull this out ? It's a good reminder for me, though, to update `index.rst`...

Reply:
Discussed with @tsalo offline and removed this